undefined reference to `increment_node_ids(long)`

I'm attempting to use libodgi and libhandlegraph together, and getting errors like this:

```
/usr/bin/ld: odgi-prefix/lib/libodgi.a(odgi.cpp.o):(.data.rel.ro._ZTCN4odgi7graph_tE208_N11handlegraph20DeletableHandleGraphE[_ZTVN4odgi7graph_tE]+0x1e8): undefined reference to `handlegraph::MutableHandleGraph::increment_node_ids(long)'
/usr/bin/ld: odgi-prefix/lib/libodgi.a(odgi.cpp.o):(.data.rel.ro._ZTCN4odgi7graph_tE208_N11handlegraph18MutableHandleGraphE[_ZTVN4odgi7graph_tE]+0x190): undefined reference to `handlegraph::MutableHandleGraph::increment_node_ids(long)'
/usr/bin/ld: odgi-prefix/lib/libodgi.a(odgi.cpp.o):(.data.rel.ro._ZTCN4odgi7graph_tE0_N11handlegraph29MutablePathMutableHandleGraphE[_ZTVN4odgi7graph_tE]+0x4e8): undefined reference to `handlegraph::MutableHandleGraph::increment_node_ids(long)'
/usr/bin/ld: odgi-prefix/lib/libodgi.a(odgi.cpp.o):(.data.rel.ro._ZTCN4odgi7graph_tE0_N11handlegraph31MutablePathDeletableHandleGraphE[_ZTVN4odgi7graph_tE]+0x518): undefined reference to `handlegraph::MutableHandleGraph::increment_node_ids(long)'
```

It looks like there is a kind of confusion about nid_t and long, and this virtual function isn't being instantiated correctly.

This can be resolved by adding a specialization of the function for long.

--- a/src/handle.cpp
+++ b/src/handle.cpp
@@ -119,6 +119,10 @@ void MutableHandleGraph::increment_node_ids(nid_t increment) {
     });
 }
 
+void MutableHandleGraph::increment_node_ids(long increment) {
+    increment_node_ids((nid_t)increment);
+}
+
 std::vector<step_handle_t> PathHandleGraph::steps_of_handle(const handle_t& handle,
                                                             bool match_orientation) const {
     std::vector<step_handle_t> found;

--- a/src/include/handlegraph/mutable_handle_graph.hpp
+++ b/src/include/handlegraph/mutable_handle_graph.hpp
@@ -85,6 +85,8 @@ public:
     /// Has a default implementation in terms of reassign_node_ids, but can be
     /// implemented more efficiently in some graphs.
     virtual void increment_node_ids(nid_t increment);
+    /// This specialization for long appears to be needed to avoid confusion about nid_t
+    virtual void increment_node_ids(long increment);
     
     /// Renumber all node IDs using the given function, which, given an old ID, returns the new ID.
     /// Modifies the graph in place. Invalidates all outstanding handles.

